Sources/ComposableArchitecture/Store.swift

```diff
@@ -345,12 +346,22 @@ public final class Store<State, Action> {
       .dropFirst()
       .sink { [weak localStore] newValue in
         guard !isSending else { return }
+
+        let callbackInfo = Instrumentation.CallbackInfo<Self.Type, Any>(storeKind: Self.self, action: nil).eraseToAny()
```

Comment:
Can we use something like `NSThread.callStackSymbols()` to get the callee's file or class at the very least? Not sure how hard that is or if it's even useful in unsymbolicated code.

Reply:
We could consider it in the browser swift code but not in TCA I think. But beyond that we already capture the store type which pretty well defines where the store is used. Do you have a specific use case in mind that ID not currently covered?

Reply:
It'd be useful to easily go from the entry in the trace to where it is in the code. But I guess you could look at a dedup right under it for that?